tools/sqlite.lua

```diff
@@ -48,13 +48,13 @@ local function factory(theProduct)
   local function create(filename)
     local dbinit = { 'BEGIN' }
     for k, v in pairs(defs) do
-      table.insert(dbinit, ('CREATE TABLE %s (%s)'):format(k, table.concat(v.orderedfields, ',')))
+      table.insert(dbinit, ('CREATE TABLE %s ("%s")'):format(k, table.concat(v.orderedfields, '","')))
```

Comment:
this is to handle column name `Index` 🙁

Reply:
oh this is a great fix. ideally it'd be in a separate PR but don't bother (yet)